.github/workflows/codeql-analysis.yml

```diff
@@ -2,7 +2,8 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ develop, master ]
+    branches: [ develop, master, 1.10.3 ]
```

Comment:
I think need to revert this before merging.

Reply:
Eventually this will go into main and will need to be rebased against main anyway there it's develop and main only.  But yeah 1.10.3 needs to be removed from here.